XLS-0033d-CFTs/README.md

```diff
@@ -834,6 +896,22 @@ MAXIMUM TOTAL SIZE:         2000 (250 bytes)
 
 TODO: Validate these numbers as they may be slightly low for trust lines. For example, in addition to the above data, trust lines require two directory entries for low/high nodes that get created for each trust line (i.e., for each RippleState object). This creates two root objects, each 98 bytes, adding 196 bytes in total per unique issuer/holder. Conversely, for CFTs, the page structure allows for up to 32 issuances to be held by a single token holder while only incurring around 102 bytes for a CFTokenPage. Thus, every time an account wants to hold a new token, the current Trustline implementation would require _at least_ 430 bytes every time. If we imagine a single account holding 20 tokens, CFTs would require ~1040 bytes, whereas trust lines would require ~8,600 bytes!
  
+# Appendix 2: `CFTokenIssuanceID` construct options
+There have been several options in how to construct the CFTokenIssuanceID:
+## Problem: Hash by issuer address and currency code (conventional approach)
+This is the conventional way of constructing an identifier, as the token holder can directly submit the transaction after knowing the issuer and the currency they want to use. However, if this is implemented, it means that after the issuer destorys/burns the token, they can re-issue the token with the same currency, resulting in the same `CFTokenIssuanceID`. This behavior can be misleading to token holders who have held onto the first iteration of the token rather than the re-issued one(since both iterations have the same `CFTokenIssuanceID`). In real world use cases(especially finance), after a fungible token is burnt, it should not be possible to re-create it, and absolutely not with the same identifier. 
```

Comment:
Could you please clarify "This behavior can be misleading to token holders who have held onto the first iteration..."? `CFTokenIssuance` can only be destroyed if there are no holders of CFT so once it's destroyed, there is tokens left from this iteration.

Reply:
Yeah I'm not too exactly sure of this either...I think Shashwat might have better idea from the product perspective.

Reply:
And this is not really different in that respect from IOU. I.e. the issuer can delete the trustline once all IOU's are redeemed and then create it again. It would be good to understand what problem we are really solving by going the uniqueid approach for each new `CFTokenIssuance` rather than using issuer/assetCode for the id.

Reply:
i agree